estate/models/estate_property.py

```diff
@@ -127,3 +128,34 @@ def _onchange_garden(self):
         else:
             self.garden_area = 0
             self.garden_orientation = False
+
+    # `self` must be a single "estate.property" record
+    def _check_availability(self, error_msg):
+        if self.state in ["sold", "canceled"]:
+            raise UserError(self.state.capitalize() + " properties cannot " + error_msg)
```

Comment:
```suggestion
           message = ("%s  properties cannot %s", self.state.capitalize(),  error_msg)
            raise UserError(message)
```

Reply:
Is this change is for the string to work with i18n? This would need a call to `gettext` (`_("")`) in the error messages.

Reply:
Solved.